app/views/iterations/_index_iteration.html.erb

```diff
@@ -17,31 +17,38 @@
             <h4 class="modal-title" id="myModalLabel">Iteration Tasks Graph</h4>
           </div>
           <div class="modal-body">
-                <div class="text-secondary">
-                  <h4>Preliminary Tasks</h4>
-                  <div class="row">
-                    <% tasks.select{|task| task.updater_type == 'preliminary'}.each do |task| %>
-                        <%= render partial: 'plain_task_card', locals: {node: task}%>
-                    <% end %>
-                  </div>
-                </div>
-                <div class="text-secondary">
-                  <h4>Development Tasks</h4>
-                  <div class="row">
-                    <% tasks.select{|task| task.updater_type == 'development'}.each do |task| %>
-                        <%= render partial: 'plain_task_card', locals: {node: task}%>
-                    <% end %>
-                  </div>
-                </div>
-                <div class="text-secondary">
-                  <h4>Post Task</h4>
-                  <div class="row">
-                    <% tasks.select{|task| task.updater_type == 'post'}.each do |task| %>
-                        <%= render partial: 'plain_task_card', locals: {node: task}%>
-                    <% end %>
-                  </div>
-                </div>
-
+            <div class="text-secondary">
```

Comment:
Again, using a dictionary reduces the repetitiveness here.

Reply:
fixed